buildSrc/build.gradle.kts

```diff
@@ -53,6 +53,14 @@ val googleAuthToolVersion = "2.1.2"
 val licenseReportVersion = "1.16"
 val grGitVersion = "3.1.1"
 
+/**
+ * The version of ErrorProne Gradle plugin.
+ *
+ * Always use the same version as the one specified in `io.spine.internal.dependency.ErrorProne`.
+ * Otherwise, when testing Gradle plugins, clashes may occur.
```

Comment:
We're digging deeper into the needs of manual verification of versions. Someday it would break. It's not the matter of “if”, it's a matter of “when”.

How can we avoid this?

@armiol, FYI.

Reply:
Well, I have little experience with Gradle but still can say what I think :)

As far as I understand `buildSrc/src/.../dependency` supplies typed versions to build files in concrete projects. `buildSrc/build.gradle.kts` also needs that versions but can't access types for assembling of which it's responsible. 

1. Straight patch solution. Versions that are also needed in `buildSrc/build.gradle.kts` are to be put in something like `buildSrc/versions.properties`. Affected types from `buildSrc/src/.../dependency` and `buildSrc/build.gradle.kts` now use that file as a source of versions;
2. Big change. Gradle has an incubating feature for [version catalogs](https://melix.github.io/blog/2021/03/version-catalogs.html).